server/odc-common/src/test/java/com/oceanbase/odc/common/util/MapUtilsTest.java

```diff
@@ -97,4 +98,37 @@ public void formatKvString_ValueContainsSeparator_IllegalArgumentException() {
         map.put("a", "2,");
         MapUtils.formatKvString(map);
     }
+
+    @Test
+    public void mapEqualsTest() {
```

Comment:
unit testing case naming follow style:
`{methodUnderTest}_{scenario}_{expectedResult}`

Reply:
split in four methods